src/students/students.service.ts

```diff
@@ -4,40 +4,106 @@ import { DBService } from 'src/database/db.service';
 import { CreateStudentDto } from './dto/create-student.dto';
 import { UpdateStudentDto } from './dto/update-student.dto';
 
+interface Iresponse {
+  status: string;
+  messages: string[];
+  data?: Student;
+}
+
```

Comment:
I propose a change here . Please wait for my next push and follow my code structure at Admins Module and Common package.

Reply:
change committed